docs/source/checks/conditional_probabilities.rst

```diff
@@ -91,47 +91,61 @@ Parameters
 
 ``min_support``:
     This parameter expects a float between 0 and 1 and impacts both runtime
-    and size of the result. :ref:`conditional_probabilities` only produces
+    and size of the result. :ref:`conditional_probabilities` only considers
     rules whose ``support_of_left_side`` and ``support`` exceed ``min_support``
     in at least one of the two data sets.
 
-    The lower you choose ``min_support`` the more resources are required
-    both in terms of memory and CPU.
+    The lower you choose ``min_support`` the more resources are required during
+    computation both in terms of memory and CPU.
     The default value is 0.01. This means that :ref:`conditional_probabilities`
     only considers values which appear in at least 1% of your tuples.
     By adjusting this parameter you can adjust the granularity of the comparison
     of the two data sets.
 
 ``min_confidence``:
     This parameter expects a float between 0 and 1 and impacts the size of the
-    result. :ref:`conditional_probabilities` only produces rules whose
+    result. :ref:`conditional_probabilities` only considers rules whose
     ``confidence`` exceeds ``min_confidence`` in at least one of the two data sets.
 
-    The lower you choose ``min_confidence`` the more rules are generated.
+    The lower you choose ``min_confidence`` the more rules are considered.
     The default value is 0.15. This means that the conditional probability
-    of a right side given a left side has to be at least 15%.
+    of a right side (consequence) given a left side (antecedent) has to be at least 15%.
 
 ``rule_limit``:
-	This parameter expects an int and controls the maximum number of rules that are
-	printed as a result of executing :ref:`conditional_probabilities`.
-	The default value is 5. This means that the 5 most significant rules are printed.
+    This parameter expects an int and controls the maximum number of rules that are
+    printed in each section of the report as a result of executing
+    :ref:`conditional_probabilities`.
+    The default value is 5. This parameter does not have any impact on the diagram.
 
 ``min_delta_supports``:
-	This parameter expects a float between 0 and 1 and affects the granularity of the
-	comparison of the two data sets. Only rules whose support values exhibit an absolute
-	difference of more than ``min_delta_supports`` are considered during computation.
-	The default value is 0.05.
+    This parameter expects a float between 0 and 1 and affects the granularity of the
+    comparison of the two data sets. Only rules whose support values exhibit an absolute
+    difference of more than ``min_delta_supports`` are considered to indicate a shift.
+    A rule has to exceed ``min_delta_supports`` to be classified as an orange rule. If it
+    also exceeds ``min_delta_confidences`` it is classiefied as a red rule.
+    The default value is 0.05. This parameter does not impact the diagram. 
```

Comment:
see comment above

Reply:
I will update the documentation in another pr late. But will keep this in mind